dist/apb.js

```diff
@@ -82,13 +83,14 @@ var apb = /** @class */ (function () {
         });
     };
     //* BOOLEAN CHECKS & Validators /////////////////////////////////////////////////////
-    apb.prototype.isStateIntegration = function (stateName, States) {
+    apb.prototype.isInteractionStateWithParameters = function (stateName, States) {
         if (States === void 0) { States = this.States; }
         if (States[stateName] === undefined) {
```

Comment:
question: We have `void 0` in one place and then `undefined` on the next line. Is there something we were hoping to accomplish with `void 0` that the next line should benefit from?